plugins/inputs/tail/tail.go

```diff
@@ -100,7 +100,7 @@ func (t *Tail) SampleConfig() string {
 }
 
 func (t *Tail) Description() string {
-	return "Stream a log file, like the tail -f command"
+	return "Parse the new lines appended to a log file"
```

Comment:
Let's mention the interval like we do for the file input.  Also I wonder if leaving out "log file" would help too.  When I hear that I think /var/log/messages and I don't think of a growing csv or line protocol or json file.

How about "Parse the lines appended to a file since the last interval".

If you agree avoiding "log file" is right, we need to remove it from the readme too.

Reply:
Agree about the "log file", we don't want people to think of this plugin is only for traditional logs.  I have some reservations about mentioning interval as this plugin doesn't use the interval.  How about: "Parse the newly appended lines to a file"?